File: CS50X/week1/pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // height or num of rows
    for (int i = 1; i <= height; i++)
    {
        // spaces before pyramid
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }

        // left pyramid
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        // gap
        printf("  ");

        // right pyramid
        for (int l = 0; l < i; l++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: CS50X/week6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

for i in range(n):
    j = i + 1
    print(" " * (n - j) + "#" * j + "  " + "#" * j)
